src/Version.cpp

```diff
@@ -34,28 +35,36 @@
 namespace espressopp {
 
   Version::Version() {
-	  name          = "ESPResSo++";
-      major         = MAJORVERSION;
-	  minor         = MINORVERSION;
-	  patchlevel    = PATCHLEVEL;
-	  gitrevision    = gitversion;
-      boostversion  = BOOST_LIB_VERSION;
-	  date          = __DATE__;
-	  time          = __TIME__;
+    name_          = "ESPResSo++";
+    major_         = MAJORVERSION;
+    minor_         = MINORVERSION;
+    patchlevel_    = PATCHLEVEL;
+    gitrevision_    = gitversion;
+    boostversion_  = BOOST_LIB_VERSION;
+    date_          = __DATE__;
+    time_          = __TIME__;
```

Comment:
I don't really have a strong opinion, but I don't like changing stuff just for the sake of indentation and style guides. @stuehn , up to you.

Maybe we should run `uncrustify` and google style checker on the whole code in a separate change.